packages/blade/src/components/Card/__tests__/Card.native.test.tsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { fireEvent } from '@testing-library/react-native';
+import {
+  CardBody,
+  Card,
+  CardFooter,
+  CardFooterLeading,
+  CardFooterTrailing,
+  CardHeader,
+  CardHeaderLeading,
+  CardHeaderTrailing,
+  CardHeaderIcon,
+  CardHeaderCounter,
+  CardHeaderBadge,
+} from '../Card';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+import { InfoIcon } from '~components/Icons';
+import { Text } from '~components/Typography';
+import { Counter } from '~components/Counter';
+import { Badge } from '~components/Badge';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<Card />', () => {
+  it('should render a plain Card', () => {
+    const { toJSON } = renderWithTheme(
+      <Card surfaceLevel={2}>
+        <CardBody>
+          <Text>Plain Card</Text>
+        </CardBody>
+      </Card>,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render a Card with Header', () => {
```

Comment:
same test suite for footer?

Reply:
There is `should render a Card with Footer` let me move it up for easy scanning.